qiskit/visualization/pulse_visualization.py

```diff
@@ -20,11 +20,11 @@
 import warnings
 from typing import Union, Callable, List, Dict, Tuple
 
-from qiskit.pulse import Schedule, Instruction, SamplePulse, ScheduleComponent
-from qiskit.pulse.channels import Channel
-from qiskit.visualization.pulse.qcstyle import PulseStyle, SchedStyle
-from qiskit.visualization.exceptions import VisualizationError
+from qiskit import pulse
+from qiskit.pulse import channels as chans
```

Comment:
Similar comment here

Reply:
Seem comments on similar issues on why I need to do this.